pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -1076,7 +1076,7 @@ func (oe *operationExecutor) CheckVolumeExistenceOperation(
 			if mounter == nil {
 				return false, fmt.Errorf("mounter was not set for a filesystem volume")
 			}
-			if isNotMount, mountCheckErr = mounter.IsLikelyNotMountPoint(mountPath); mountCheckErr != nil {
+			if isNotMount, mountCheckErr = mount.IsNotMountPoint(mounter, mountPath); mountCheckErr != nil {
```

Comment:
@jingxu97 Is it OK to change from `IsLikelyNotMountPoint` to `IsNotMountPoint` from performance perspective? We may see performance improvement soon as commented [here](https://github.com/kubernetes/kubernetes/issues/79980#issuecomment-1079081306), but not yet and not all enviornments.

Reply:
I'm wondering the same honestly, or if there's a better approach. It solves the problem in the simplest possible case, but I can't say I understand all the performance implications. Seems generally better to be correct than fast, but the timeouts in `pull-kubernetes-e2e-gce-storage-disruptive` look concerning.

Reply:
We do this only once on kubelet startup though

Reply:
I think the impact of this will be fairly minimal. In addition to the points above from @gnufied (we only do this once on kubelet startup) and @mkimuram (about the upcoming performance improvement), `IsNotMountPoint` still attempts to use `IsLikelyNotMountPoint` first, before falling back to the `/proc/mounts` check:
https://github.com/kubernetes/kubernetes/blob/05b59e77175a87b7caca8b27a3ed2478df3d452e/staging/src/k8s.io/mount-utils/mount.go#L230-L239
So this should not affect drivers for which `IsLikelyNotMountPoint` works today. But it will allow drivers like csi-hostpath which use bind mounts to work correctly.

Reply:
to speed up IsNotMountPoint check, this is the pending PR https://github.com/kubernetes/kubernetes/pull/109217

I think it is ok to use IsNotMountPoint here